### ISSUE TITLE
Add support to convert/parse go code to Pseudo AST?

### ISSUE THREAD

**Issue by pombredanne** [alehander42/pseudo#18]
... thsi would allow to some of porting from Go to Python, etc,...

**Comment by alehander42** [comment]
yep, actually that would allow to port/compile some Go code to all supported targets(Python, JS, C#, Ruby etc)

Basically one needs to create a Go->Pseudo compiler, similar to [pseudo-python](https://github.com/alehander42/pseudo-python)

I am not a huge fan of Go, so that's probably not very high on my priority list, but it's not something complicated (Go is statically typed, so the compiler would be way simpler than pseudo-python which does auto type inference)

**Comment by Zireael07** [comment]
bump, go seems easy to parse and is a commonly used language nowadays
